bin/check-bundle-size.js

```diff
@@ -41,6 +41,7 @@ const tooBigs = files.filter(f => f.tooBig);
 if (tooBigs.length) {
   log('!!!');
   log(`!!! ${tooBigs.length} file(s) are too big !!!`);
+  log(`!!! Files: ${tooBigs.map(f => f.path).join('; ')}`);
```

Comment:
Logs in Github doesn't show colors, so it is hard to see which files are too big

Reply:
Good catch; can you link to build logs where this occurs?

Reply:
If logging here, maybe:

```suggestion
  log(`!!! ${tooBigs.length} file(s) are too big:`);
  log('!!!');
  tooBigs.map(f => log(`!!! * f.path`));
  log('!!!');
```

?

Otherwise this could be fixed by renaming `redBold()` e.g. `flagViolation()` and e.g.

```js
if(enableColours) return [ RED, BOLD, text, RESET ].join('');
else return `${text}\t<-- TOO BIG`;
```

or something.